tests/components/reolink/test_host.py

```diff
@@ -97,11 +97,12 @@ async def test_webhook_callback(
     assert config_entry.state is ConfigEntryState.LOADED
 
     webhook_id = config_entry.runtime_data.host.webhook_id
+    unique_id = config_entry.runtime_data.host.unique_id
 
     signal_all = MagicMock()
     signal_ch = MagicMock()
-    async_dispatcher_connect(hass, f"{webhook_id}_all", signal_all)
-    async_dispatcher_connect(hass, f"{webhook_id}_0", signal_ch)
+    async_dispatcher_connect(hass, f"{unique_id}_all", signal_all)
+    async_dispatcher_connect(hass, f"{unique_id}_0", signal_ch)
```

Comment:
I added assertions for the entity state such that we check that the callback indeed reaches the entity.

However I would like to keep the specific connections to the dispatcher, since it is more clear to check for `assert_not_called` in that way (as opposed to a state that did not change)